helm/templates/controller/role.yaml

```diff
@@ -19,45 +20,21 @@ rules:
 - apiGroups:
   - sbombastic.rancher.io
   resources:
+  - images/finalizers
   - registries/finalizers
   verbs:
   - update
 - apiGroups:
   - sbombastic.rancher.io
   resources:
-  - registries/status
-  verbs:
-  - get
-  - patch
-  - update
-- apiGroups:
-  - sbombastic.sbombastic.rancher.io
-  resources:
-  - images
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - sbombastic.sbombastic.rancher.io
-  resources:
-  - images/finalizers
-  verbs:
-  - update
-- apiGroups:
-  - sbombastic.sbombastic.rancher.io
-  resources:
```

Comment:
I'm just confused, why these rules have been dropped?

Reply:
because they were wrong: 
`sbombastic.sbombastic.rancher.io` instead of `sbombastic.rancher.io`